src/openrct2/drawing/Sprite.cpp

```diff
@@ -177,6 +177,34 @@ void mask_scalar(sint32 width, sint32 height, const uint8 * RESTRICT maskSrc, co
     }
 }
 
+static std::string gfx_get_csg_header_path()
+{
+    auto path = Path::ResolveCasing(Path::Combine(gConfigGeneral.rct1_path, "Data", "csg1i.dat"));
+    if (path.empty())
+    {
+        path = Path::ResolveCasing(Path::Combine(gConfigGeneral.rct1_path, "RCTdeluxe_install", "Data", "csg1i.dat"));
+    }
+    return path;
+}
+
+static std::string gfx_get_csg_data_path()
+{
+    // csg1.1 and csg1.dat are the same file.
+    // In the CD version, it's called csg1.1 on the CD and csg1.dat on the disk.
+    // In the GOG version, it's always called csg1.1.
+    // In the Steam version, it's always called csg1.dat.
+    auto path = Path::ResolveCasing(Path::Combine(gConfigGeneral.rct1_path, "Data", "csg1.1"));
+    if (path.empty())
+    {
+        path = Path::ResolveCasing(Path::Combine(gConfigGeneral.rct1_path, "Data", "csg1.dat"));
+        if (path.empty())
+        {
+            path = Path::ResolveCasing(Path::Combine(gConfigGeneral.rct1_path, "RCTdeluxe_install", "Data", "csg1.1"));
```

Comment:
In the comment, it says this file is always called csg1.dat in Steam. Not csg1.1.